src/components/Wellcome.js

```diff
@@ -4,23 +4,23 @@ import dogBackgroundImg from '../images/dog-background.jpg'
 import Header from './Header'
 import Button from './common/Button'
 
-const Wellcome = () => (
+const Wellcome = ({ setMenuVisible }) => (
     <Wrapper>
         <BackgroundImage>
-            <HeightDefinerDiv/>
+            <HeightDefinerDiv />
         </BackgroundImage>
         <ConentWrapper>
-            <Header/>
+            <Header setMenuVisible={setMenuVisible} />
             <TitleWrapper>
                 <Title>
                     Find your Best Friend
                 </Title>
                 <Text>
-                    This site is a perfect place to find a perfect pet.<br/>
-                    Animals bring a lot of light and joy in our life.<br/>
-                    Pick one.<br/>
-                    Save one live.<br/>
-                    Find a new love.<br/>
+                    This site is a perfect place to find a perfect pet.<br />
```

Comment:
FYI: you can make a new component and pass text as a reference
anyway I don't think BR scales well on multiple screen and with any length of text